src/Compilers/VisualBasic/Portable/BoundTree/BoundNodes.xml

```diff
@@ -722,6 +722,7 @@
      (a: 1, b: (c: 1, d: null))  does not have a natural type, because "null" does not have one
     -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="allow"/>
+    <Field Name="InferredType" Type="TupleTypeSymbol" Null="allow"/>
```

Comment:
Agree. Given we have two types here, that will be very helpful.

---
In reply to: [76694399](https://github.com/dotnet/roslyn/pull/13335#discussion_r76694399) [](ancestors = 76694399)